src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        int figureNumber = 11;
+
+        int figuresNumberInSetOne = 0;
+
+        if (figureNumber % 2 == 0) {
+            figuresNumberInSetOne = figureNumber / 2;
+        } else {
+            figuresNumberInSetOne = figureNumber / 2 + 1;
+        }
+
+        Figure [] result = new Figure[figureNumber];
+        FigureSupplier figureSupplier = new FigureSupplier();

```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. It's good that you've instantiated `FigureSupplier` before the loop starts.